resources/views/contacts/contactUs.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layout')
+
+@section('content')
+    <div class="row">
+        <div class="col-lg-12">
+            <h4>Contact us</h4>
+            @if ($errors->any())
+                <div class="alert alert-danger">
+                    <ul>
+                        @foreach ($errors->all() as $error)
+                            <li>{{ $error }}</li>
+                        @endforeach
+                    </ul>
+                </div>
+            @endif
+            @if (session()->has('success'))
+                <div class="alert alert-success">
+                    {{ session('success') }}
+                </div>
+            @endif
+            <form action="{{ route('contactUs.send') }}" method="POST" name="contact-form">
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="email">Email</label>
+                        <input type="email" value="{{ old('email') }}" required="required" class="form-control" name="email" id="email" placeholder="example@example.com">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="name">Full Name</label>
+                        <input type="text" value="{{ old('name') }}" required="required" class="form-control" name="name" id="name" placeholder="John Doe">
+                        @error('name')
```

Comment:
I think it is extra, you already have block with all errors

Reply:
Is it better to have an error block for each separate input? Isn't it more user friendly?

Reply:
if you removed the main block with errors, then yes. You can keep it.